test/config/treepathtest.go

```diff
@@ -70,4 +74,13 @@ func (s *TestSuite) TestTreePath(t *testing.T) {
 
 	//  Make sure new root got removed
 	gnmiutils.CheckGNMIValue(ctx, t, gnmiClient, getPath, gnmiutils.NoExtensions, "", 0, "New root was not removed")
+
+	// Make sure path got removed
+	interfacesPath := gnmiutils.GetTargetPath(simulator.Name(), newConfigPath)
```

Comment:
I think you should be checking the root path - to see if this interface got deleted
In gnmi it's like having no elem in the query - so I guess in test it would be like the path `/`